src/main/java/windebot/Winde.java

```diff
@@ -0,0 +1,123 @@
+     *
+     * @param filePath The file path for storing task history.
+     */
+
+    Winde(String filePath) {
+        //ui = new Ui();
+        history = new History(filePath);
+        reminder = new Reminder(history.load());
+        commandType = "";
+        willContinue = true;
+    }
+
+    /**
+     * Constructs a Winde object with the default file path for storing history.
+     */
+
+    public Winde() {
+        //ui = new Ui();
+        history = new History();
+        reminder = new Reminder(history.load());
+        commandType = "";
+    }
+
+    /**
+     * The main method that starts the WindeBot application.
+     *
+     * @param args Command-line arguments (not used).
+     */
+/*
+    public static void main(String[] args) {
+        new Winde().run();
+    }
+
+ */
+
+    /**
+     * Starts the main loop of the WindeBot application.
+     */
+/*
+    private static void run() {
+        ui.greet();
+        Command currentCommand = new ListCommand();
+        boolean shouldContinue = true;
+        while (shouldContinue) {
+            try {
+                String input = ui.read();
+                ui.showLine();
+                currentCommand = Parser.parse(input);
+                shouldContinue = currentCommand.execute(input, reminder, ui);
+            } catch (UnsupportedCommandException e) {
+                throw new RuntimeException(e);
+            } catch (EmptyDescriptionException e) {
+                throw new RuntimeException(e);
+            } catch (TooManyParametersException e) {
+                throw new RuntimeException(e);
+            } finally {
+                ui.showLine();
+            }
+        }
+        currentCommand.exit(history, reminder, ui);
+    }
+
+ */
+
+    public String getResponse(String input) {
+        try {
+            Command currentCommand = Parser.parse(input);
+            Ui ui = new Ui();
+            willContinue = currentCommand.execute(input, reminder, ui, history);
+            commandType = currentCommand.whatCommand();
+            return ui.getOutput();
+        } catch (UnsupportedCommandException e) {
+            throw new RuntimeException(e);
+        } catch (EmptyDescriptionException e) {
+            throw new RuntimeException(e);
+        } catch (TooManyParametersException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public String getCommandType() {
+        return commandType;
+    }
+
+    public String hello() {
+        String output = "__        __              _" + "\n";
+        output += "\\ \\      / /(_) _ __   __| | ___ " + "\n";
+        output += " \\ \\ /\\ / / | || '_  \\/ _` |/ _ \\ " + "\n";
+        output += "  \\ V  V /  | || | | || (_ || __/ " + "\n";
+        output += "   \\_/\\_/   |_||_| |_|\\__,_|\\___| " + "\n";
+        output += "Hello! I'm Winde\n" + "What can I do for you?" + "\n";
+        return output;
+    }
```

Comment:
I like the ASCII art!

Reply:
Thanks! I spent like 1.5hrs on it haha

Reply:
hahaha there are ascii art generators online!